contracts/modules/token/TokenController.sol

```diff
@@ -36,6 +36,10 @@ contract TokenController is ITokenController, LockHandler, LegacyMasterAware {
     claimsReward = claimsRewardAddress;
   }
 
+  function unlistClaimsReward() external {
+    token.removeFromWhiteList(claimsReward);
```

Comment:
added an issue to manage this in master instead

Reply:
any particular reason to do this in master and not here?

Reply:
I guess there are multiple contracts that require this